scripts/jenkins/set-node-version.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Install nvm for this shell
+source ~/.nvm/nvm.sh
+
+# Use node.js at 4.2.x
+nvm install 4.2
```

Comment:
`4` would be enough, the latest `4.x.y` release is an LTS, it so happens it's `4.2.y` in fact.